packages/grid/test/sorting.test.js

```diff
@@ -220,6 +245,39 @@ describe('sorting', () => {
       expect(sorterLast._order).to.equal(null);
     });
 
+    describe('multi-sort-on-shift-click=true', () => {
```

Comment:
```suggestion
    describe('multi-sort-on-shift-click', () => {
```

Reply:
Or actually, `multiSortOnShiftClick` is better because that's the name of the property

Reply:
Done.